_posts/2017-07-30-nonbinary-tech.md

```diff
@@ -0,0 +1,23 @@
+---
+layout:     post
+title:      What we learned at nonbinary.tech
+date:       2017-07-30 12:00
+summary:    This weekend the first ever nonbinary in tech event took place in London, and I had the privilege of attending.
+category:   Community
+tags:       lgbt, career
+author:     alice_kaerast
+---
+
+This weekend I had the privilege of attending [nonbinary.tech](https://nonbinary.tech), the world's first conference highlighting the amazing work of those of us who identify outside of the gender binary in the technology industry.  The event took place at Goldsmiths University in London in a rather nice accessible venue.  The talks and discussion concentrated on the community side of technology, because pure technology alone is easy to navigate.
+
+**The Sound of Many Sleeping Bees** was a great talk on how phone notifications take us out of the presence of the moment, and how they can be like rude personal assistants who are constantly interrupting us.  They seem like a living thing, and by design we are encouraged to repeatedly interact with them.  I found myself comparing the annoying notifications of terrible apps to comments from people who haven't yet unlearned cissexism - each time it happens isn't bad enough to break any rules or code of conduct, but both are terribly draining.
+
+**Diversity by Default in Design** was given by a designer about their efforts to show diversity in their artwork, and the battles with companies to actually get that artwork used.  Eriol talked about how "edge-case" has become a word to discourage time being spent on making products accessible to marginalized groups and how making products friendly is universal, not just for the dominant user group.  Stock images for minority groups can throw up some worrying results, illiciting negative stereotypes, and it is important to remember that just because a stock image exists doesn't mean it is acceptable to use.  If you are in a position to represent your product users through photography, use photographs of actual users; and if those photographs don't show diversity then maybe you have bigger problems to address first.
```

Comment:
hi richard @mitchellrj  . how are those google play reviews doing. seems you like live casino. i am quite impressed by your skills and information security. everyone at big g00gles is impressed too . are those reviews still there. what kind of servers is in your data center? oh ok i found some.